src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String str : records) {
+            String[] temp = str.split(":"); //Подскажите бест-практис, пожалуйста
```

Comment:
your solution is ok, but fix naming 
```suggestion
        for (String record : records) {
            String[] splitedRecord = str.split(":"); //Подскажите бест-практис, пожалуйста
```